lib/ansible/modules/cloud/azure/azure_rm_aks_facts.py

```diff
@@ -33,6 +33,19 @@
     tags:
         description:
             - Limit results by providing a list of tags. Format tags as 'key' or 'key:value'.
+    show_kubeconfig:
+        description:
+            - Show kubeconfig of the AKS cluster.
+            - Note the operation will cost more network overhead, not recommond to use when listing AKS.
+        version_added: 2.8
+        choices:
+            - user
+            - admin
+    show_available_versions_in_location:
+        description:
+            - Show available kubernetes versions in a specific location.
+            - Note, When this field set, nothing will be queried in your subscription but only available kubernetes versions.
```

Comment:
i don't quite understand what it means.
does it mean no instances will be listed?
if so then perhaps this should be in a separate module?